environments/staging/public.yml

```diff
@@ -12,6 +12,9 @@ kafka_scala_version: 2.13
 kafka_inter_broker_protocol_version: 2.6
 kafka_log_message_format_version: 2.6
 
+zookeeper_version: 3.7.0
+zookeeper_cluster: False
```

Comment:
I would have thought this would be `True` for a multi-node setup

Reply:
Yes, Danny. I will make that change. Probably when I was testing the False state I missed it changing back. Will fix it. thanks